src/lib/Scenes/City/CitySavedList.tsx

```diff
@@ -75,7 +75,7 @@ export default createPaginationContainer(
       }
       me {
         followsAndSaves {
-          shows(first: $count, status: CURRENT, city: $citySlug, after: $cursor)
+          shows(first: $count, status: CURRENT, dayThreshold: 30, city: $citySlug, after: $cursor)
```

Comment:
The [CI failure](https://travis-ci.org/artsy/emission/jobs/506013251) is on this line. It's a legit failure; the schema doesn't support this variable here.

Reply:
I just PR'd that - https://github.com/artsy/metaphysics/pull/1583

Reply:
Nice!